pkg/bufferdecoder/decoder.go

```diff
@@ -61,7 +61,7 @@ func (decoder *EbpfDecoder) DecodeContext(ctx *Context) error {
 	_ = copy(ctx.UtsName[:], decoder.buffer[offset+76:offset+92])
 	ctx.Flags = binary.LittleEndian.Uint32(decoder.buffer[offset+92 : offset+96])
 	ctx.EventID = events.ID(int32(binary.LittleEndian.Uint32(decoder.buffer[offset+96 : offset+100])))
-	// offset 100:103 is used for padding
+	ctx.OrigSyscall = binary.LittleEndian.Uint32(decoder.buffer[offset+100 : offset+104])
```

Comment:
This conflicts with @geyslan multiscope work (both are touching the padding). @yanivagman FYIO. Since Geyslan work has priority, I believe you will have to change the approach here.